### ISSUE TITLE
2017.2 CLI Activation

### ISSUE THREAD

**Issue by brianmxwll** [nenadg/docker-unity3d#2]
You mention using --username, --password,  and --serial to activate via the command line in Unity >= 2017.2. Do you have some documentation or examples of this? As far as I can tell, it isn't currently possible.

https://forum.unity.com/threads/5-4-0p1-fails-to-launch-due-to-license.425277/#post-3265724

**Comment by nenadg** [comment]
As far as I tried you can't register 'Personal edition' in this fashion, haven't tried licensed version though.

**Comment by brianmxwll** [comment]
Sounds like you don't have that use case, but if you ever try and succeed... please pass it along. Doesn't appear to be possible at this point in time with any license type.

**Comment by nenadg** [comment]
No, I don't. I mentioned it in comment above as I tried (and failed due to lack of serial number) that method. Your link to forum post explains it better.
